packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -261,6 +280,18 @@ export class GumbyController {
                 reason: err?.code,
                 reasonDesc: getTelemetryReasonDesc(err),
             })
+
+            // TODO: if build system version is available at this stage, include buildSystemVersion field.
```

Comment:
Local build system version is not available here, but we do already emit a metric for this when that logic runs: https://github.com/aws/aws-toolkit-vscode/blob/3c8c3a12a16042d5a8eb61afee3f8afc849efe5a/packages/core/src/codewhisperer/commands/startTransformByQ.ts#L120-L126

Reply:
I'm planning to deprecate `isDoubleClickedToTriggerInvalidProject` with `validateProject`, but thanks for sharing! I'll see if I can get the information earlier.

Reply:
I tried fetching maven version in an earlier step, but got `'undefined (mvn)'`. I need more investigation here, but since we weren't publishing maven version in project validation metrics before, I'll skip it in this PR.